protoc-gen-twirp/generator.go

```diff
@@ -1067,6 +1067,13 @@ func pathPrefix(file *descriptor.FileDescriptorProto, service *descriptor.Servic
 // pathFor returns the complete path for requests to a particular method on a
 // particular service.
 func pathFor(file *descriptor.FileDescriptorProto, service *descriptor.ServiceDescriptorProto, method *descriptor.MethodDescriptorProto) string {
+	return pathPrefix(file, service) + method.GetName()
+}
+
+// pathForCamelCompat returns the old CamelCase complete path for requests to a
+// particular method on a particular service.
+// This method is here to ensure compatibility with older clients.
```

Comment:
This comment isn't quite accurate, since clients will still make requests with camel-cased paths. This ultimately should link to documentation around this issue (which doesn't exist just yet).

I need to poke some folks internally about the proper place for these docs, and then we can update this comment to direct users to it.

Reply:
I made #262, which will add a page to the docs about this known issue.